php/ext/google/protobuf/package.xml

```diff
@@ -750,5 +750,20 @@ G  A release.
    <notes>
    </notes>
   </release>
+  <release>
+   <version>
+    <release>3.15.0RC1</release>
+    <api>3.15.0</api>
+   </version>
+   <stability>
+    <release>beta</release>
+    <api>beta</api>
+   </stability>
+   <date>2021-02-05</date>
+   <time>14:15:36</time>
+   <license uri="https://opensource.org/licenses/BSD-3-Clause">3-Clause BSD License</license>
+   <notes>
```

Comment:
I wasn't able to update the script in time, could we paste in the actual PHP changes here before releasing?

Doesn't have to be in this PR, especially if we haven't put the release notes for PHP in `CHANGES.txt` yet.